tests/gcp/installer/util/test_resource/forseti_conf_server.yaml.in

```diff
@@ -0,0 +1,244 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
```

Comment:
I'm pretty sure we only need this boiler plate on code, not configurations or sample files. Not that it hurts anything, but it would be cleaner to read the sample file without.

Reply:
Removed in both client/server yaml.in and yaml.sample.